test/compare_output.zig

```diff
@@ -444,4 +444,18 @@ pub fn addCases(cases: &tests.CompareOutputContext) {
 
         tc
     });
+
+    cases.addCase("windowsLoadDll failure",
```

Comment:
When possible, a behavior test is preferred over a compare_output test, because the behavior tests run much faster. See test/behavior.zig and test/cases/*